sw/airborne/modules/computer_vision/active_random_filter.cpp

```diff
@@ -0,0 +1,1959 @@
+/*
+ * Copyright (C) Wilco Vlenterie (wv-tud)
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+/**
+ * @file "modules/computer_vision//cv_active_random_filter.c"
+ * @author Wilco Vlenterie (wv-tud)
+ * Active random sampling colour filter
+ */
+
+#include "active_random_filter.h"
+#include <vector>
+#include <ctime>
+
+#define BOARD_CONFIG "boards/bebop.h"       // TODO: WHYYYY?
```

Comment:
good question, why is this needed ?
if should be something like `#include BOARD_CONFIG` in place of the `#include "boards/bebop.h"` below and removing this line